stubs/protobuf/google/protobuf/text_format.pyi

```diff
@@ -0,0 +1,204 @@
+import sys
+from _typeshed import SupportsWrite
+from typing import Any, Callable, Iterable, Optional, Text, Tuple, TypeVar, Union
+
+from .descriptor import FieldDescriptor
+from .descriptor_pool import DescriptorPool
+from .message import Message
+
+if sys.version_info < (3, 0):
+    long = int
+
+_M = TypeVar("_M", bound=Message)  # message type (of self)
+
+class Error(Exception): ...
+
+class ParseError(Error):
+    def __init__(self, message: Optional[str] = ..., line: Optional[int] = ..., column: Optional[int] = ...) -> None: ...
+    def GetLine(self) -> Optional[int]: ...
+    def GetColumn(self) -> Optional[int]: ...
+
+class TextWriter:
+    def __init__(self, as_utf8: bool) -> None: ...
+    if sys.version_info < (3, 0):
```

Comment:
You can just unconditionally uses Text; in Python 2 str/bytes is an implicit subtype of Text.